src/Benchmarks/Data/EfDb.cs

```diff
@@ -26,19 +26,33 @@ public Task<World> LoadSingleQueryRow()
             return _dbContext.World.FirstAsync(w => w.Id == id);
         }
 
-        public async Task<World[]> LoadMultipleQueriesRows(int count)
+        public async Task<World[]> LoadMultipleQueriesRows(int count, bool withTracking = false)
         {
             var result = new World[count];
 
             for (int i = 0; i < count; i++)
             {
                 var id = _random.Next(1, 10001);
-                result[i] = await _dbContext.World.FirstAsync(w => w.Id == id);
+                var world = withTracking ? _dbContext.World.AsTracking() : _dbContext.World;
+                result[i] = await world.FirstAsync(w => w.Id == id);
             }
 
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count, true);
+
+            for (int i = 0; i < count; i++)
+            {
+                result[i].RandomNumber = _random.Next(1, 10001);
+            }
+
+            await _dbContext.SaveChangesAsync();
```

Comment:
So, we are currently in violation of the spec because we have a single transaction around all writes. Easiest fix right now is to move SaveChanges inside the loop, which gets us to a transaction per iteration. Disabling transactions completely will require injecting a custom service; let me know if you want to proceed with that and I can figure out how to do it.